From cb46e9f18ca6254c78f76000f28479be41aa522c Mon Sep 17 00:00:00 2001
From: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Date: Sun, 17 Sep 2023 11:52:28 +0900
Subject: [PATCH] hwdb: ieee1394-unit-function: add Weiss Engineering DAC202
 (Maya edition)

Michele Perrone and Ralf Anderegg contribute to ALSA dice driver to support
products of Weiss Engineering. Their patch includes support for DAC202 Maya
edition.

* https://lore.kernel.org/alsa-devel/24703333-9250-40bf-e736-a5f3c4862034@weiss.ch/

This commit fulfulls an entry for the model as well as supplemental comment
to DAC2/Minerva.

(cherry picked from commit 042cfb896055f52a77b7d91156920ef4105b569d)

Related: RHEL-5950
---
 hwdb.d/80-ieee1394-unit-function.hwdb | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/hwdb.d/80-ieee1394-unit-function.hwdb b/hwdb.d/80-ieee1394-unit-function.hwdb
index 33bf7d93d8..503f5bbc14 100644
--- a/hwdb.d/80-ieee1394-unit-function.hwdb
+++ b/hwdb.d/80-ieee1394-unit-function.hwdb
@@ -903,7 +903,7 @@ ieee1394:ven00001C6Amo00000002sp00001C6Aver00000001
 ieee1394:node:ven0x001c6amo0x000003units0x001c6a:0x000001
 ieee1394:ven00001C6Amo00000003sp00001C6Aver00000001
  ID_VENDOR_FROM_DATABASE=Weiss Engineering
- ID_MODEL_FROM_DATABASE=Minerva
+ ID_MODEL_FROM_DATABASE=DAC2/Minerva
  IEEE1394_UNIT_FUNCTION_AUDIO=1
 
 ieee1394:node:ven0x001c6amo0x000004units0x001c6a:0x000001
@@ -924,6 +924,12 @@ ieee1394:ven00001C6Amo00000007sp00001C6Aver00000001
  ID_MODEL_FROM_DATABASE=DAC202
  IEEE1394_UNIT_FUNCTION_AUDIO=1
 
+ieee1394:node:ven0x001c6amo0x000008units0x001c6a:0x000001
+ieee1394:ven00001C6Amo00000008sp00001C6Aver00000001
+ ID_VENDOR_FROM_DATABASE=Weiss Engineering
+ ID_MODEL_FROM_DATABASE=DAC202/Maya
+ IEEE1394_UNIT_FUNCTION_AUDIO=1
+
 #
 # Digidesign Digi00x family.
 #
